modules/ModulesManager/Http/Routes/admin.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+# This is the admin path. Comment this out if you don't have an admin panel component.
+Route::get('/', 'AdminController@index');
+Route::post('/addModule', 'AdminController@addModule')->name('addModule');
```

Comment:
To keep these with the conventions being used:

- Use `GET /add` and `POST /add` for adding/removing
- Editing `GET /{id}` to edit; and `POST /{id}` to submit it
- Deleting:  `DELETE /{id}`

Also needs: `->middleware('ability:admin')` to restrict it